### PR TITLE
feat(icons): Watson discovery icons

### PR DESCRIPTION
**Pull request by chrisconnors-ibm**
#### Changelog

**New**

- adds 
32/improve-relevance.svg
32/phrase-sentiment.svg
32/folders.svg
32/exit.svg
32/document-sentiment.svg
32/character-patterns.svg

**Changed**

- updates metadata.yml and categories.yml

#### Testing / Reviewing

{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}

### CODE REVIEW

**Comment by chrisconnors-ibm** [comment]
cc @conradennis

File: packages/icons/metadata.yml [hunk 1]
```diff
@@ -336,6 +336,15 @@ icons:
         categories:
           - name: Actions
             subcategory: Navigation
+  - name: character-patterns
+    friendly_name: Character patterns
+    usage: This is a description for usage
+    categories:
+      - name: Organization
+        subcategory: Data
+    aliases:
```
Comment (vpicone) [inline packages/icons/metadata.yml:345]: You gotta either specify aliases or leave this property off all together.

**Review by vpicone — CHANGES_REQUESTED** [review]
Are we trying to use appropriate usage data or still sticking with "This is a description for usage"

**Review by jnm2377 — COMMENTED** [review]
LGTM once Vince's comment is addressed regarding alias.

**Comment by chrisconnors-ibm** [comment]
> Are we trying to use appropriate usage data or still sticking with "This is a description for usage"

I don't think we've populated that anywhere yet. future property

File: packages/icons/metadata.yml (hunk 1 above)
Comment (chrisconnors-ibm) [reply packages/icons/metadata.yml:345]: removed that 'aliases' prop. Thanks Vince.

**Review by vpicone — APPROVED** [review]

**Review by laurenmrice — APPROVED** [review]
looks good !

**Review by joshblack — APPROVED** [review]

**Pull request merged**